src/Tizen.NUI/src/public/NUIWidgetApplication.cs

```diff
@@ -42,7 +42,6 @@ public NUIWidgetApplication( System.Type widgetType ) : base(new NUIWidgetCoreBa
         /// The constructor for multi widget class and instance.
         /// </summary>
         /// <param name="widgetTypes">List of derived widget class type.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public NUIWidgetApplication(Dictionary<System.Type, string> widgetTypes) : base(new NUIWidgetCoreBackend())
```

Comment:
is this constructor to support multiple widget class? Not clear the title & description of this patch.